app/controllers/court_dates_controller.rb

```diff
@@ -53,6 +53,12 @@ def update
     end
   end
 
+  def destroy
+    return unless current_user.casa_admin? || current_user.supervisor?
```

Comment:
This should use the policy file

Reply:
https://github.com/rubyforgood/casa/blob/main/app/policies/court_date_policy.rb

Reply:
i.e. `authorize @court_date` or similar

Reply:
Thanks for your review. I'll take a look at this later today and comeback.

Reply:
I made some changes.